.github/workflows/smoketest-python37-v4.yml

```diff
@@ -17,6 +17,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Run V2 Python 3.7 Smoke Test
+    - name: Run V4 Python 3.7 Smoke Test
```

Comment:
I was recently informed that [Python 3.7 has reached EOL](https://devguide.python.org/versions/). Should we update this to a newer Python version?

Reply:
I don't feel strongly, but I like testing with a minimal version of Python just so that we try (within reason) to prevent breaking customers where possible. It's still a useful signal knowing that our SDKs continue working in old versions of Python.